reporter-web-app/src/components/SummaryView.js

```diff
@@ -352,6 +352,10 @@ class SummaryView extends React.Component {
                                                 tab={(
                                                     <span>
                                                         Violations (
+                                                        {
+                                                            ruleViolations.length !== unresolvedRuleViolations
+                                                            && `${unresolvedRuleViolations}/`
```

Comment:
Nit: Maybe spaces around `/`? Similar below.

Reply:
I actually tried a version with spaces but it looked better without and o smaller screens the 2 spaces make the difference whether or not all tabs columns are shown.